pom.xml

```diff
@@ -166,6 +165,59 @@
                             </arguments>
                         </configuration>
                     </execution>
+                    <!-- Build extension -->
+                    <execution>
```

Comment:
Interesting, so you actually already call npm from maven, so probably could also call a typescript script which inlines the script

Reply:
Actually we call the shell script during `npm run build` (see `package.json`) ;)